roc/config.go

```diff
@@ -379,8 +380,8 @@ type SenderConfig struct {
 	// The samples written to the sender are buffered until the full packet is
 	// accumulated or the sender is flushed or closed. Larger number reduces
 	// packet overhead but also increases latency.
-	// If zero, default value is used.
-	PacketLength uint64
+	// If zero, default value is used. If negative, the error is returned.
```

Comment:
I suggest to rephrase this:

> If negative, the error is returned.

to:

> Should not be negative, otherwise an error is returned.

To make it clear that negative values are not desired usage.